### PR TITLE
Added solution for Project Euler problem 85.

### PR DESCRIPTION
**Pull request by fpringle**
Name: Counting rectangles

By counting carefully it can be seen that a rectangular grid measuring 3 by 2 contains eighteen rectangles:
￼
Although there exists no rectangular grid that contains exactly two million rectangles, find the area of the grid with the nearest solution.

Reference: [https://projecteuler.net/problem=85](url)
Reference: #2695

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by dhruvmanila — COMMENTED** [review]
### Announcement:

@fpringle

*This message is sent to all pull requests which are **OPEN, NOT MERGED, not containing 'wontfix' label and contains 'Project Euler' or 'Project' or 'Euler' (case doesn't matter) in their title**. If this message doesn't apply to your pull request, please ignore this.*

### Message:

This is to notify all the users submitting a pull request for the Project Euler solution that the [README.md](https://github.com/TheAlgorithms/Python/blob/master/project_euler/README.md) file has been updated with all the style requirements necessary for the directory and solution files. ***Please read through it and make all the necessary changes required to your submission.***

**Review by 53jk1 — APPROVED** [review]

File: project_euler/problem_085/sol1.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    72
+    """
+    triangle_numbers = [0]
+    for i in range(1, ceil(sqrt(target * 2) * 1.1)):
+        triangle_numbers.append(triangle_numbers[-1] + i)
+
+    best = 0
+    area = 0
+
+    for a, tri_a in enumerate(triangle_numbers[1:], 1):
```
Comment (dhruvmanila) [inline project_euler/problem_085/sol1.py:29]: Descriptive variable names would be awesome in the long run!
No idea what is `a` or `tri_a`. What does `hit` even mean? What is `best`?

Comment (fpringle) [reply project_euler/problem_085/sol1.py]: Good point, fixed.

**Comment by dhruvmanila** [comment]
Open a new PR or rebase on the latest commit.

**Pull request closed without merging**